src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -320,7 +320,7 @@ func onBootstrapControlPlane(s *state.State, initConfig map[string]string) error
 	}
 
 	if err := client.WaitApiServerReady(s.Context); err != nil {
-		return fmt.Errorf("k8s api server did not become ready in time: %w", err)
+		return fmt.Errorf("kube api-server did not become ready in time: %w", err)
```

Comment:
the service name is `kube-apiserver`
```suggestion
		return fmt.Errorf("kube-apiserver did not become ready in time: %w", err)
```